### PR TITLE
fix(react-skeleton): Stories should import from react-components

### PR DESCRIPTION
**Pull request by behowell**
## Previous Behavior

Skeleton stories import directly from individual packages: react-skeleton and react-field.

## New Behavior

Change Skeleton stories to import from react-components instead of individual packages.

## Related Issue(s)

Similar fix in react-progress: #27497

### CODE REVIEW

**Comment by fabricteam** [comment]
## 📊 Bundle size report

<details>
<summary>Unchanged fixtures</summary>

| Package & Exports                                                                                                                                                             |          Size (minified/GZIP) |
| ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ----------------------------: |
| <samp>react-components</samp> <br /> <abbr title='bundle-size/ButtonProviderAndTheme.fixture.js'>react-components: Button, FluentProvider & webLightTheme</abbr>              |  `67.994 kB`<br />`18.395 kB` |
| <samp>react-components</samp> <br /> <abbr title='bundle-size/MultipleComponents.fixture.js'>react-components: Accordion, Button, FluentProvider, Image, Menu, Popover</abbr> | `207.512 kB`<br />`57.555 kB` |
| <samp>react-components</samp> <br /> <abbr title='bundle-size/ProviderAndTheme.fixture.js'>react-components: FluentProvider & webLightTheme</abbr>                            |  `38.819 kB`<br />`12.351 kB` |
| <samp>react-portal-compat</samp> <br /> <abbr title='bundle-size/PortalCompat.fixture.js'>PortalCompatProvider</abbr>                                                         |    `6.446 kB`<br />`2.185 kB` |

</details>
<sub>🤖 This report was generated against <a href='https://github.com/microsoft/fluentui/commit/763202a8995b7ca3e9e552828e07f015ffbdc6d7'>763202a8995b7ca3e9e552828e07f015ffbdc6d7</a></sub>
<div id="bundleSizeComment9423"/>

**Comment by fabricteam** [comment]
## [Perf Analysis (`@fluentui/react-components`)](https://github.com/microsoft/fluentui/wiki/Perf-Testing)

  <table>
  <tr>
    <th>Scenario</th>
    <th>Render type</th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">Master Ticks</a>
    </th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">PR Ticks</a>
    </th>
    <th>Iterations</th>
    <th>Status</th>
  </tr><tr>
            <td>InfoButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27498/perf-test-react-components/InfoButton-mount_base.html">21</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27498/perf-test-react-components/InfoButton-mount.html">21</a></td>
            <td>5000</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27498/perf-test-react-components/InfoButton-mount.regression.txt">Possible regression</a></td>
           </tr></table><details><summary>All results</summary><p>
  <table>
  <tr>
    <th>Scenario</th>
    <th>Render type</th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">Master Ticks</a>
    </th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">PR Ticks</a>
    </th>
    <th>Iterations</th>
    <th>Status</th>
  </tr><tr>
            <td>Avatar</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27498/perf-test-react-components/Avatar-mount_base.html">690</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27498/perf-test-react-components/Avatar-mount.html">694</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Button</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27498/perf-test-react-components/Button-mount_base.html">366</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27498/perf-test-react-components/Button-mount.html">362</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Field</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27498/perf-test-react-components/Field-mount_base.html">1241</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27498/perf-test-react-components/Field-mount.html">1239</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProvider</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27498/perf-test-react-components/FluentProvider-mount_base.html">897</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27498/perf-test-react-components/FluentProvider-mount.html">903</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27498/perf-test-react-components/FluentProviderWithTheme-mount_base.html">119</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27498/perf-test-react-components/FluentProviderWithTheme-mount.html">119</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27498/perf-test-react-components/FluentProviderWithTheme-virtual-rerender_base.html">103</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27498/perf-test-react-components/FluentProviderWithTheme-virtual-rerender.html">96</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27498/perf-test-react-components/FluentProviderWithTheme-virtual-rerender-with-unmount_base.html">107</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27498/perf-test-react-components/FluentProviderWithTheme-virtual-rerender-with-unmount.html">119</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>InfoButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27498/perf-test-react-components/InfoButton-mount_base.html">21</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27498/perf-test-react-components/InfoButton-mount.html">21</a></td>
            <td>5000</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27498/perf-test-react-components/InfoButton-mount.regression.txt">Possible regression</a></td>
           </tr>
<tr>
            <td>MakeStyles</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27498/perf-test-react-components/MakeStyles-mount_base.html">1183</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27498/perf-test-react-components/MakeStyles-mount.html">1124</a></td>
            <td>50000</td>
            <td></td>
           </tr>
<tr>
            <td>Persona</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27498/perf-test-react-components/Persona-mount_base.html">2020</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27498/perf-test-react-components/Persona-mount.html">1918</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SpinButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27498/perf-test-react-components/SpinButton-mount_base.html">1619</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27498/perf-test-react-components/SpinButton-mount.html">1563</a></td>
            <td>5000</td>
            <td></td>
           </tr></table></p></details>

<div id="perfComment9425"/>

**Comment by fabricteam** [comment]
## 🕵 fluentuiv9 No visual regressions between this PR and main
<div id="vrtCommentfluentuiv9"/>

File: packages/react-components/react-progress/stories/ProgressBar/index.stories.tsx [hunk 1]
```diff
@@ -1,4 +1,4 @@
-import { ProgressBar } from '@fluentui/react-progress';
+import { ProgressBar } from '@fluentui/react-components';
```
Comment (behowell) [inline packages/react-components/react-progress/stories/ProgressBar/index.stories.tsx:1]: Missed this in the react-progress PR #27497. Sneaking it in here...

File: packages/react-components/react-skeleton/stories/Skeleton/SkeletonAnimation.stories.tsx [hunk 2]
```diff
@@ -1,7 +1,6 @@
 import * as React from 'react';
-import { Skeleton, SkeletonItem, SkeletonProps } from '@fluentui/react-skeleton';
-import { Field } from '@fluentui/react-field';
 import { makeStyles, tokens } from '@fluentui/react-components';
+import { Field, Skeleton, SkeletonItem, SkeletonProps } from '@fluentui/react-components/unstable';
```
Comment (spmonahan) [inline packages/react-components/react-skeleton/stories/Skeleton/SkeletonAnimation.stories.tsx:3]: Nit: should be `import type { SkeletonProps } from ...`

File: packages/react-components/react-skeleton/stories/Skeleton/SkeletonAppearance.stories.tsx [hunk 3]
```diff
@@ -1,7 +1,6 @@
 import * as React from 'react';
-import { Skeleton, SkeletonItem, SkeletonProps } from '@fluentui/react-skeleton';
-import { Field } from '@fluentui/react-field';
 import { makeStyles, tokens } from '@fluentui/react-components';
+import { Field, Skeleton, SkeletonItem, SkeletonProps } from '@fluentui/react-components/unstable';
```
Comment (spmonahan) [inline packages/react-components/react-skeleton/stories/Skeleton/SkeletonAppearance.stories.tsx:3]: Ditto

File: packages/react-components/react-skeleton/stories/Skeleton/SkeletonRow.stories.tsx [hunk 4]
```diff
@@ -1,6 +1,6 @@
 import * as React from 'react';
-import { Skeleton, SkeletonItem, SkeletonProps } from '@fluentui/react-skeleton';
 import { makeStyles, shorthands, tokens } from '@fluentui/react-components';
+import { Skeleton, SkeletonItem, SkeletonProps } from '@fluentui/react-components/unstable';
```
Comment (spmonahan) [inline packages/react-components/react-skeleton/stories/Skeleton/SkeletonRow.stories.tsx:3]: Ditto

**Review by spmonahan — APPROVED** [review]

**Pull request merged**